pysat/instruments/pysat_testing.py

```diff
@@ -177,6 +187,8 @@ def load(fnames, tag='', inst_id='', sim_multi_file_right=False,
 
     if tag == 'default_meta':
         return data, pysat.Meta()
+    elif tag == 'no_download':
+        return pds.DataFrame(), pysat.Meta()
     else:
         return data, meta
```

Comment:
Seems like we could speed this up by having the if/else logic nest the definition of `data` and `meta` rather than putting them here at the end.  Also ok opening this up as an issue (probably best to add a TODO too in that case, to make this easier to find later).

Reply:
documented as #1120